app/templates/events/view/tickets/attendees.hbs

```diff
@@ -0,0 +1,76 @@
+<div class="ui hidden divider"></div>
+<div class="ui compact menu pull right floated">
```

Comment:
@anu-007 The tickets/attendees tab and tickets/orders are identical UIs, so I think they should follow a general schema. Can you please make the layout  like the tickets/orders so that you use grid instead of compact menu twice.@geekyd Do you agree?